fix select arg detection

Reference:
 https://bugs.gentoo.org/921487

Upstream: N/A, unresponsive contact email address.

Signed-off-by: Yegor Yefremov <yegorslists@googlemail.com>
--- a/autoconf_local/aclocal.m4
+++ b/autoconf_local/aclocal.m4
@@ -4196,7 +4196,7 @@
 #include <sys/select.h>
 #endif
 #include <sys/socket.h>	/* MG: always incl */
-extern select ($ac_cv_func_select_arg1,$ac_cv_func_select_arg234,$ac_cv_func_select_arg234,$ac_cv_func_select_arg234,$ac_cv_func_select_arg5);],,dnl
+extern int select ($ac_cv_func_select_arg1,$ac_cv_func_select_arg234,$ac_cv_func_select_arg234,$ac_cv_func_select_arg234,$ac_cv_func_select_arg5);],,dnl
         [ac_not_found=no ; break 3],ac_not_found=yes)
       done
      done
